doc/changes/latest.inc

```diff
@@ -27,6 +27,7 @@ Enhancements
 
 Bugs
 ~~~~
+- Extended test to highlight bug in :func:`mne.stats.permutations.permutation_t_test` (:gh:`11575` by :newcontrib:`Joshua Calder-Travis`)
```

Comment:
```suggestion
- Extended test to highlight bug in :func:`mne.stats.permutations.permutation_t_test` (:gh:`11575` by :newcontrib:`Joshua Calder-Travis`_)
```

There is a missing underscore for sphinx to create the link.

Reply:
I have added it. Thanks @mscheltienne